harness/propertyHelper.js

```diff
@@ -75,14 +62,27 @@ function verifyProperty(obj, name, desc, options) {
     "The desc argument should be an object or undefined, " + String(desc)
   );
 
+  var names = Object.getOwnPropertyNames(desc);
+  for (var i = 0; i < names.length; i++) {
+    assert(
+      names[i] === "value" ||
+        names[i] === "writable" ||
+        names[i] === "enumerable" ||
+        names[i] === "configurable" ||
+        names[i] === "get" ||
+        names[i] === "set",
+      "Invalid descriptor field: " + names[i],
+    );
+  }
+
   var failures = [];
 
   if (Object.prototype.hasOwnProperty.call(desc, 'value')) {
     if (!isSameValue(desc.value, originalDesc.value)) {
       failures.push("descriptor value should be " + desc.value);
     }
     if (!isSameValue(desc.value, obj[name])) {
-      failures.push(failures, "object value should be " + desc.value);
+      failures.push("object value should be " + desc.value);
     }
```

Comment:
This whole block seems unnecessary and potentially a source of incorrect failures, but I guess that's just the state of `verifyProperty`.